pkg/ui/styl/components/select.styl

```diff
@@ -15,3 +17,13 @@
     display inline-block
     vertical-align middle
     width 200px
+
```

Comment:
Should we just move all of them into `shame.styl`, or alternately a `select-shame.style` file so they're all together but still called out? I don't feel like putting all the select styles into the shame file is using shame correctly. My understanding is that it's more for "quick hacks" and things you intend to fix, but this feels a little more substantial and potentially long term (I don't like it but I don't intend to come back and fix it soon.)